manuscript/03.results.md

```diff
@@ -18,13 +18,53 @@ We chose to prioritize CMTF results that were able to predict infection outcome
 
 ### CMTF-based data integration improves predictions of patient outcomes
 
-Text.
-
-![**CMTF-based data integration improves predictions of patient outcomes.** A) Prediction accuracy when using the cytokines or gene expression measurements on their own, or when combined using CMTF. Accuracy was assessed by cross-validation within the training cohort. B) Prediction ROC curve based on cross-validation within the training cohort. C) ROC curve from validation cohort. C) Prediction accuracy using all pairs of components from CMTF. D) The prediction decision function with respect optimal pair of components for prediction.](figure3.svg "Figure 3"){#fig:figure3 width="100%"}
+To assess the integration of data via CMTF, we developed a logistic regression model to predict MRSA persistence via 
+subject factors. Prior to the unblinding of the validation cohort, we evaluated the accuracy of our CMTF model using a 
+10-fold cross-validation over our training cohort ({@fig:figure3}A). To evaluate CMTF's improvement in outcome prediction, we also
+developed logistic regression models to predict MRSA persistence using serum cytokine, serum IL-10, plasma cytokine, and
+plasma IL-10 measurements. We find that CMTF produces a balanced accuracy score of 0.70, slightly below the 
+balanced accuracy score of the best-performing logistic regression model trained solely against serum cytokine data.
```

Comment:
I've been thinking about this comparison a lot (before your writing here)... I think it might not be fair. We know CMTF performs worse on subjects without plasma cytokines. Wouldn't it be more fair to compare the performance of CMTF and the serum cytokine data only on the subjects with serum cytokines? This also applies to the ROC curves (which is what got me thinking about it).

Reply:
I agree--it's not really a fair comparison. I'll re-run those accuracy measurements so that the CMTF is only scored against serum/plasma samples.